src/Translator/src/DependencyInjection/UxTranslatorExtension.php

```diff
@@ -35,7 +35,24 @@ public function load(array $configs, ContainerBuilder $container)
         $loader = (new PhpFileLoader($container, new FileLocator(\dirname(__DIR__).'/../config')));
         $loader->load('services.php');
 
-        $container->getDefinition('ux.translator.translations_dumper')->setArgument(0, $config['dump_directory']);
+        $dumperConfigDefinition = $container->getDefinition('ux.translator.translations_dumper.configuration');
+        $dumperConfigDefinition->setArgument(0, $config['dump_directory']);
+
+        $webpackDumperDefinition = $container->getDefinition('ux.translator.translations_dumper.webpack');
+        $webpackDumperDefinition->setArgument(0, $config['dump_directory']);
+
+        $assetMapperDumperDefinition = $container->getDefinition('ux.translator.translations_dumper.asset_mapper');
+        // Setting to the default value because the folder isn't configurable with asset mapper
+        $assetMapperDumperDefinition->setArgument(0, $this->processConfiguration($configuration, [])['dump_directory']);
```

Comment:
I think we should use some configuration for the user to chose which service he wants to use.. 
... and alias it as "TranslationDumper".